Include in missing typedefs needed in order to build DAQ.

---  daq-0.6.2/sfbpf/sll.h.orig	2010-05-06 15:13:26.000000000 -0400
+++  daq-0.6.2/sfbpf/sll.h	2012-01-13 16:27:21.000000000 -0500
@@ -82,6 +82,15 @@
 #define SLL_HDR_LEN	16          /* total header length */
 #define SLL_ADDRLEN	8           /* length of address field */
 
+#if defined (__SVR4) && defined (__sun)
+
+typedef uint8_t u_int8_t;
+typedef uint16_t u_int16_t;
+typedef uint32_t u_int32_t;
+
+#endif /* defined (__SVR4) && defined (__sun) */
+
+
 struct sll_header
 {
     u_int16_t sll_pkttype;      /* packet type */
